app/controllers/samples_controller.rb

```diff
@@ -138,11 +138,25 @@ def index_v2
                  current_power.samples
                end
 
+    # Add total_reads
+    # TODO: (gdingle): this appears to be making n+1 queries
+    @samples = @samples.left_joins(:pipeline_runs).select("*, pipeline_runs.total_reads, pipeline_runs.adjusted_remaining_reads")
+
     respond_to do |format|
       format.json do
         render json: @samples.offset(offset).limit(limit).as_json(
-          only: [:id, :name, :sample_tissue, :host_genome_id, :project_id, :created_at],
-          methods: []
+          only: [
+            :id,
+            :name,
+            :sample_tissue,
+            :host_genome_id,
+            :project_id,
+            :created_at,
+            :sample_location,
+            :total_reads,
+            :adjusted_remaining_reads
+          ],
+          methods: [:host_genome_name]
```

Comment:
flagging this as potentially inefficient (unless `includes(:host_genome)` is added), because it calls the `host_genome` model

Reply:
Please make sure it doesn't do a query on host_genomes for each sample

Reply:
Thanks for the improvement. I agree this needs to be at least as fast as before.

Reply:
Fixed with help from @cdebourcy